canopeum_backend/canopeum_backend/views.py

```diff
@@ -726,12 +791,9 @@ def post(self, request, postId):
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
     @extend_schema(responses={201: LikeSerializer}, operation_id="like_delete")
-    def delete(self, request, postId):
+    def delete(self, request: Request, postId):
         try:
             post = Post.objects.get(pk=postId)
-        except Post.DoesNotExist:
-            return Response(status="sef")
```

Comment:
I don't think `"sef"` status was intended? I went with `HTTP_404_NOT_FOUND` by combining the try-except